pom.xml

```diff
@@ -19,7 +19,7 @@
 		<dcsa.events.version>0.0.47</dcsa.events.version>
 		<dcsa.core.tag/>
 		<dcsa.event.tag/>
-		<dcsa.artifacttype>-SNAPSHOT</dcsa.artifacttype>
+		<dcsa.artifacttype></dcsa.artifacttype>
```

Comment:
Any reason for removing `-SNAPSHOT` from artifacttype?

Reply:
SNAPSHOT reference is back